manifest.json

```diff
@@ -8,7 +8,7 @@
     },
     {
       "address": "14egjs9cSxt4jnaZTYQE4NemQVXL6qtayqNNpy3VJPSS6AAcaUq",
-      "signature": ""
+      "signature": "nVXH4O1qpUGrP1pNQWMSLjLfYyLxK28Y9B7YdDG8mncnKKWs+tw4Y0FZpY39gOGLLMYCA2EtVeD6j0RdF/3NCQ=="
```

Comment:
this doesn't look like a valid signature.

Reply:
Sig was on the wrong address. Checks have now passed.